ci/run.sh

```diff
@@ -26,7 +26,8 @@ helm install projectriff/istio --name istio --namespace istio-system --devel --w
   --set gateways.istio-ingressgateway.type=${K8S_SERVICE_TYPE}
 helm install projectriff/riff --name riff --devel \
   --set riff.runtimes.core.enabled=true \
-  --set riff.runtimes.knative.enabled=true
+  --set riff.runtimes.knative.enabled=true \
+  --set cert-manager.enabled=false
```

Comment:
a `--wait` would also help since we can no longer create resources until the web hooks are ready